src/data/defaults.txt

```diff
@@ -97,6 +97,10 @@ global	macroLens	false
 global	mementoListActive	false
 global	mergeHobopolisChat	false
 global	pingLatest
+global	pingLogin	true
```

Comment:
Do we want this to default to true and deal with users who wonder what happened or do we default it to false and figure users knew what they were doing when they flipped it?  Either way I like the fact that if pingLogin is true what happens with the defaults will be reasonable for most users, most of the time.

Reply:
pingLogin simply says "do a ping test and save the results" - in pingLatest, pingLongest, and pingShortest".
It adds 10 api.php requests to logging in - less than 300 msec, in Boston.

If the user later decides to use the other features - enabling pingLoginCheck and setting pingLoginGoal or pingLoginThreshold, as appropriate - they will have the historical data to help them make an informed decision.

Perhaps pingLoginCheck should default to "none", and the user will have to select it.

Reply:
Yeah, I did that. We will collect ping data by default, but the user will have to set pingLoginCheck to goal or threshold, as desired, on the Connection tab of the Login Frame.